test/etaoin/api_test.clj

```diff
@@ -647,16 +657,28 @@
 
 (deftest test-switch-window-next
   (let [init-handle (e/get-window-handle *driver*)]
-    (doseq [_ (range 3)]
+    (dotimes [n 3]
```

Comment:
Right... `dotimes`, nice.

Reply:
Yea, if you're doing it a specific number of times, it's better to use `dotimes`. It's just semantics, but slightly better semantics.